app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1911,11 +1909,33 @@ BrowserWorld::CreateSkyBox(const std::string& aBasePath, const std::string& aExt
 
   if (extension == ".ktx") {
 #if defined(OPENXR) && defined(OCULUSVR)
-    glFormat =  GL_COMPRESSED_SRGB8_ETC2;
+    glFormat = GL_COMPRESSED_SRGB8_ETC2;
 #else
-    glFormat =  GL_COMPRESSED_RGB8_ETC2;
+    glFormat = GL_COMPRESSED_RGB8_ETC2;
 #endif
   }
+
+#if defined(OPENXR)
```

Comment:
Id rather avoid the ifdef here, since GetAvailableColorFormat will return an empty list for non-OPENXR backends.